README.md

```diff
@@ -1,44 +1,233 @@
+|MAGICPL_MAINZELLISTE_CENTRAL_API_KEY|You need to ask the central patientlists admin for this.|The apiKey for your machine to communicate with the central patientlist|
+|MAGICPL_CENTRAL_API_KEY|You need to ask the central controlnumbergenerator admin for this.|The apiKey for your machine to communicate with the central controlnumbergenerator|
+|MAGICPL_OIDC_CLIENT_ID||The client id used for your machine, to connect with the central authentication service|
+|MAGICPL_OIDC_CLIENT_SECRET||The client secret used for your machine, to connect with the central authentication service|
+
+### Cooperatively Managed Secrets
+> TODO: Describe secrets from site-config 
+
+## Managing your Bridgehead
+
+> TODO: Rewrite this section (restart, stop, uninstall, manual updates)
+There two methods to start the bridgehead. For Windows, Linux and Mac OS you can use the start-bridgehead.sh to deploy it wit docker-compose. If will also check some other setting of your system.
 
-### DKTK
+The second methods is using the systemd management tool you start, stop and update your bridgehead.
 
-For DKTK set in the site.conf the project to "dkkt". Also you need to set many settings in the env file. For the API keys for the psuenomisation you need to contact the Mainzelliste Team. 
+Just run the install-bridgehead and thats it.
 
-### GBA/BBMRI-ERIC
+## Migration Guide
+> TODO: How to transfer from windows/gbn
 
-For an GBN/BBMRI-ERIC deployment set the project to gbn. When you already deployed a bridgehead you can reuse the env file for it.
+## Pitfalls
+### [Git Proxy Configuration](https://gist.github.com/evantoli/f8c23a37eb3558ab8765)
+Unlike most other tools, git doesn't use the default proxy variables "http_proxy" and "https_proxy". To make git use a proxy, you will need to adjust the global git configuration:
 
-### C4
+``` shell
+git config --global http.proxy http://<your-proxy-host>:<your-proxy-port>;
+git config --global https.proxy http://<your-proxy-host>:<your-proxy-port>;
+```
+> NOTE: Some proxies may require user and password authentication. You can adjust the settings like this: "http://<your-proxy-user>:<your-proxy-user-password>@<your-proxy-host>:<your-proxy-port>".
+> NOTE: It is also possible that a proxy requires https protocol, so you can replace this to.
 
-For C4 project it is similar to DKTK. Set the ldm_base_url in the configuration table to null.
+You can check that the updated configuration with
 
-### Git repository
+``` shell
+git config --global --list;
+```
 
-If you already have a git config repositpory you can clone it with 
+Please repeat this configuration for the root user to. Otherwise the update service won't be able to properly update the git repository.
 
-git submodule add -f https://"$git_username":"$git_access_token"@code.mitro.dkfz.de/scm/bd/"$site_name_lowercase"-config.git ./site-config
+### Docker Daemon Proxy Configuration
```

Comment:
I'm not sure if this also needed, however setting proxy settings through https://docs.docker.com/network/proxy/ was also helpful for downloading the images